scripts/opam-env.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+eval $(opam env)
```

Comment:
this file doesn't seem to be called from anywhere

Reply:
Removed the file.